util/stream_dataset.py

```diff
@@ -391,7 +400,7 @@ def _insert_data(dataset, param, data, provenance_metadata=None, request_id=None
             # remove obs dimension from parameter's dimensions and data (13025 AC2)
             param_dimensions.remove('-obs')
             dims = param_dimensions
-            data = data[0]
+            data = data[0] if data else None
```

Comment:
the fix is split into a separate commit so that it can be back-ported into the release for sprint 12 as well as made available for the current development (ie master) branch.  I'll work on the branch in pull request 44 and get the appropriate release notes update etc, then merge that into this branch.